app/models/data_import.rb

```diff
@@ -0,0 +1,15 @@
+class DataImport < ApplicationRecord
+  has_one_attached :file
+
+  belongs_to :user
+
+  validate :file_presence
+
+  private
+
+  def file_presence
+    unless file.attached?
+      errors.add(:file, "Must upload file")
+    end
+  end
```

Comment:
I tested this in the UI, and it doesn't seem that our file validation is working.

Reply:
@littleforest I added the error message into the views and it looks like the data import is not being created unless a file is attached